week2/Q1.py

```diff
@@ -0,0 +1,44 @@
+class BinaryTree:
+	'''Data structure, consists of root and two children which are trees as well (can be empty)
+	self.root -- root element value
+	self.left -- left child value
+	self.right -- right child value
+	addRightChild -- adds right child tree
+	addLeftChild -- adds left child tree'''
+	def __init__(self,key):
+		self.left = None
+		self.right = None
+		self.root = key
+
+	def addRightChild(self,key):
+		self.right = BinaryTree(key)
+		
+	def addLeftChild(self,key):
+		self.left = BinaryTree(key)
+
+
+
+def get_ancestors(tree, key):
+	'''Returns ancestors of one key (not including itself).
```

Comment:
You can use the style of docstring in the example I shared for a template.You want to make sure you mention what it does, the arguments it takes, and the value it returns. Using "Args:" and "Results:" makes that easier to understand.